prbt_pg70_support/config/pg70.srdf.xacro

```diff
@@ -36,7 +36,7 @@ limitations under the License.
     <group name="gripper">
       <joint name="${prefix}gripper_finger_left_joint" />
     </group>
-    <end_effector group="gripper" name="pg70_end_effector" parent_link="${prefix}flange">
+    <end_effector group="gripper" name="pg70_end_effector" parent_link="${prefix}gripper_finger_left_link">
```

Comment:
Pretty much every tf but the flange seems to work, so id suggest to rahter use gripper_palm or tcp instead of the left finger link.